fabric-recipe-api-v1/src/main/java/net/fabricmc/fabric/impl/recipe/ingredient/builtin/NbtIngredient.java

```diff
@@ -145,7 +145,7 @@ public void write(JsonObject json, NbtIngredient ingredient) {
 			json.addProperty("strict", ingredient.strict);
 
 			if (ingredient.nbt != null) {
-				json.add("nbt", NbtOps.INSTANCE.convertTo(JsonOps.INSTANCE, ingredient.nbt));
```

Comment:
This wont pass checkstyle.

Reply:
That I can fix, I did the PR from web UI 😄